google/cloud/bigtable/data/_metrics/handlers/stdout.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from google.cloud.bigtable.data._metrics.handlers._base import MetricsHandler
+from google.cloud.bigtable.data._metrics.data_model import CompletedOperationMetric
+
+
+class StdoutMetricsHandler(MetricsHandler):
+    """
+    Prints a table of metric data after each operation, for debugging purposes.
```

Comment:
Please mark this as internal, we dont want our customers to enable this

Reply:
Fixed. And let me know if this is worth keeping around, I mostly made this to help with development, but could be useful for debugging in the future. I'm fine either way

FWIW though, all of this is within the private _metrics folder, so we should be able to consider it all private. Let me know if I should change that